LICENSE

```diff
@@ -97,16 +97,16 @@ distribution (with or without modification), making available to the
 public, and in some countries other activities as well.
 
   To "convey" a work means any kind of propagation that enables other
-parties to make or receive copies.  Mere interaction with a user through
+parties to make or receive copies.  Mere interaction with a creator through
 a computer network, with no transfer of a copy, is not conveying.
 
-  An interactive user interface displays "Appropriate Legal Notices"
+  An interactive creator interface displays "Appropriate Legal Notices"
```

Comment:
is there a reason why you are using "creator" instead of "user"??

Reply:
this was an accident